### ISSUE TITLE
Log in ok but game does not load

### ISSUE THREAD

**Issue by edrkns** [ChildrenOfUr/cou-issues#2328]
I am stuck on the choose your name and profile screen. The system knows me and I have logged in before. Have tried with several browsers so could someone help me please? Not sure if I should leave my email here on Git but my username is natsumi. Please help. Thanks.
![capture](https://cloud.githubusercontent.com/assets/3751766/25530576/ca964b84-2c37-11e7-9cbf-5b9809cd92b9.PNG)

**Comment by Klikini** [comment]
I'm so sorry, I'm only seeing this now because we don't get notifications for issues posted here without going through the in-game bug reporter or our email.

Is this still an issue?

**Comment by edrkns** [comment]
Have you taken the site down? I was unable to log in but now I see a 406
error. Do you have an offline version I could install and use on my own?

Thanks so much

Nat

**Comment by Klikini** [comment]
@edrkns The client relies on a program that is no longer supported in Chrome and some other browsers. We have replaced it and we are currently testing our new solution. In the meantime, could you please try playing in Firefox if you aren't already?

As for an offline version: the client can run from a computer, but it isn't so easy to run the server, database, and host all the assets on your own.

**Comment by edrkns** [comment]
Ah, okay. Thanks so much. I used vivaldi and chrome. i will try firefox.
Thanks so much. I know it's been many years but will you bring back houses,
plots, and furniture, sometime if possible?

Nat

**Comment by Klikini** [comment]
Eventually we would like to re-implement all of those features, unfortunately, our resources are very limited at the moment so it may be a while before we get to do that. Sorry :(

**Comment by edrkns** [comment]
Yes, I understand. Just glad that something is around and cubimals too 😇
do you know of any other games that could be similar?

**Comment by Klikini** [comment]
- The [Eleven Project](http://www.elevengiants.com) is working with more of the original Glitch code but might not be easy to get into.
- [MiniGlitch](https://drive.google.com/file/d/0B6E04z1fsOtCZ3QtdldjcVBYRG8/view?usp=drivesdk) is an offline street explorer. I can't find the original website, the link is to the version from 2015 that I had downloaded.

*CoU is not affiliated with and does not officially endorse any of the above projects. These are my personal suggestions.*

- [CAT422](https://github.com/revdancatt/CAT422-glitch-location-viewer) is similar to MiniGlitch but runs in a browser and has chat. Our game is actually based on this project. Unfortunately, it doesn't seem to be online anymore.

**Comment by edrkns** [comment]
I still can't get it and it's been quite a while? Do you have houses now by
any chance?
